_data/utilities.yml

```diff
@@ -169,8 +169,11 @@ websites:
 
   - name: Dominion Energy
     url: https://www.dominionenergy.com/
-    twitter: DominionEnergy
     img: dominion.png
+    tfa:
+      - sms
+      - email
+    doc: https://www.dominionenergy.com/privacy
```

Comment:
I don't think the privacy page is useful for our documentation link. The screenshots you provided are much more helpful in this regard.
```suggestion
```